strax/dtypes.py

```diff
@@ -103,6 +103,48 @@ def record_dtype(samples_per_record=DEFAULT_RECORD_LENGTH):
 ]
 
 
+def hitlet_dtype(n_widths=11, n_sample=0):
+    """
+    Hitlet dtype same as peaklet or peak dtype but for hit-kind of
+    objects.
+
+    Note:
+        If n_samples is set to zero returns an array without "data"-field.
+    """
+    hitlet_dtype = interval_dtype + [
```

Comment:
Don't you overwrite the function ```hitlet_dtype``` here

Reply:
No, since `hitlet_dtype` is just defined locally within `hitlet_dtype`, but pycharm is complaining about the style. So I will change it, thanks

```